source/code-generation/query-converter-test-bench.txt

```diff
@@ -10,10 +10,43 @@ Query Converter Test Bench
    :depth: 1
    :class: singlecol
```

Comment:
[Taxonomy]

Sorry I missed this earlier but could we add the taxonomy tags here too?

Reply:
Thank you!